app/pages/lab-fem/workflow.cjsx

```diff
@@ -0,0 +1,888 @@
+          </p>
+
+          <hr />
+
+          {if 'worldwide telescope' in @props.project.experimental_tools
+            <div>
+              <div className="deprecated">
+                <AutoSave resource={@props.workflow}>
+                  <span className="form-label">Use World Wide Telescope API</span><br />
+                  <small className="form-help">Allow user to view subject in the WWT after classifying.</small>
+                  <br />
+                  <label htmlFor="world_wide_telescope_summary">
+                    <input type="checkbox" onChange={@handleSetWorldWideTelescope} checked={@telescopeValue()}/>
+                    WorldWide Telescope
+                  </label>
+                </AutoSave>
+              </div>
+
+              <hr />
+
+            </div>}
+
+          <div className={if @isThereNotADefinedTask() then 'disabled-section' else ''}>
+            {if not @isThereNotADefinedTask()
+              <a
+                href={@workflowLink()}
+                className="standard-button"
+                target="from-lab"
+                onClick={@handleViewClick}
+              >
+                Test this workflow
+              </a>}
+            {if @isThereNotADefinedTask()
+              <div>
+                <span className="standard-button">Test this workflow</span>
+                <p>You need to add a task and content to be able to test this workflow.</p>
+              </div>}
+          </div>
+
+          <hr />
+
+          <div className={disabledIfLive}>
+            <small>
+              <button type="button" className="minor-button" disabled={@state.deletionInProgress} data-busy={@state.deletionInProgress || null} onClick={@handleDelete}>
+                Delete this workflow
+              </button>
+            </small>{' '}
+            {if @state.deletionError?
+              <span className="form-help error">{@state.deletionError.message}</span>}
+          </div>
+        </div>
+
+        <div className={taskEditorClasses}>
+          {if @state.selectedTaskKey? and @props.workflow.tasks[@state.selectedTaskKey]?
+            TaskEditorComponent = taskComponents[@props.workflow.tasks[@state.selectedTaskKey].type]?.Editor
+            <div>
+              {if 'shortcut' in @props.project.experimental_tools
+                <ShortcutEditor workflow={@props.workflow} task={@props.workflow.tasks[@state.selectedTaskKey]}>
+                  <TaskEditorComponent
+                    workflow={@props.workflow}
+                    task={@props.workflow.tasks[@state.selectedTaskKey]}
+                    taskPrefix="tasks.#{@state.selectedTaskKey}"
+                    project={@props.project}
+                    onChange={@handleTaskChange.bind this, @state.selectedTaskKey}
+                  />
+                </ShortcutEditor>
```

Comment:
I think ok to delete here, and related new component file app/pages/lab-fem/_classifier/tasks/shortcut/editor.jsx

Reply:
That's a good point, Mark - the ShortcutEditor is locked behind an experimental_tool anyway, so it should be safe to delete altogether instead of keeping it around (in such a messy form) just to have the "deprecated" text on it.